recipes/cloud-data-connector/meta.yaml

```diff
@@ -0,0 +1,56 @@
+package:
+  name: cloud-data-connector
+  version: 1.0.2
+
+source:
+  url: https://files.pythonhosted.org/packages/a2/0a/ff0e586e7cbd0fc23dec2cd9e28397a0fda71b4a52160a44b25fa9f1bbb2/cloud_data_connector-1.0.2.tar.gz
+  sha256: 7851dec0e26594361a81ead40bf536f8d3ab1e53f233c350b9eb54ac06d36311
+
+build:
+  skip: True  # [win or osx or py<39]
```

Comment:
Why not noarch?

Reply:
I was skeptical about finding all dependencies across all OS, but I will give it a try, thanks

Reply:
Even if the deps are not present, noarch is noarch, we let the solver prevent the installation in that case. That means we not only save precious CI time but also, when the missing dependencies in the other OSes is added, this package would be ready without any modification required.

Reply:
so, in the case of cloud-data-connector which has 2 dependencies that are not currently in conda (azureml and azureml-core), should I add them, or should I comment them?

Reply:
If they are mandatory you should add them. BTW, isn't `azureml` in this PR?

Reply:
yes, azureml is in this PR, I commented it and my plan was to add it from pip once cloud-data-connector was installed.
But from what you are saying, there should be another way. Do you mind guiding me in this process?

Reply:
Just uncomment it here and conda-build will do its job. You should add all the missing dependencies in this PR.

Reply:
I uncommented `azureml` and it works just fine.
For `azureml-core`, I got the following error when trying to execute `grayskull pypi --strict-conda-forge azureml-core`: `AttributeError: There is no sdist package on pypi for azureml-core` So I will keep it commented.